package.json

```diff
@@ -272,7 +261,7 @@
         "StartupWMClass": "Session"
       },
       "asarUnpack": "node_modules/spellchecker/vendor/hunspell_dictionaries",
-      "target": ["deb", "rpm", "freebsd"],
+      "target": ["deb"],
```

Comment:
I guess the CI builds for other platforms? I understand why we probably only want to build debs locally but I suggest you update the building.md to show commands on how to build for other platforms locally (passing the correct electron builder flags) for people who build from source.

Reply:
fixed